tests/test_strategies.py

```diff
@@ -237,7 +237,10 @@ def test_cold_start_parallel_tree_search(self):
         player.initialize_game()
         self.assertEqual(0, player.root.N)
         self.assertFalse(player.root.is_expanded)
-        player.tree_search(parallel_readouts=4)
+        leaves = player.tree_search(parallel_readouts=4)
+        self.assertEqual(4, len(leaves))
+        self.assertEqual(player.root, leaves[0])
```

Comment:
actually all 4 elements of leaves will be the root  - could verify that too :D

Reply:
I removed self.assertEqual(1, len(set(leaves))) because It was getting a little silly.